### DISCUSSION TITLE
Temporarily disable SKIP processing

### DISCUSSION THREAD

**Discussion by hepspl** [skiptools/skip#127] [Q&A] [answered]
Is there a temporary way to disable SKIP activity during Xcode build?
Let's say that I am working on some function which I develop only for iOS (or I plan to compatibilize with Android later) - every build operation runs SKIP plugin + runs the app in Android emulator.
For some time iOS simulator will be enough and so I would like to save time and processing power.

Is there an easy way/switch which disables SKIP for a limited time?

So far the only slight help I found is changing 
SKIP_ACTION to `build` in xcconfig file.

**Comment by marcprux** [comment, marked as answer]
We generally recommend using the "SKIP_ZERO" technique to enable/disable Skip conditionally. This is a simple flag in the `Package.swift` that allows you to turn on and off the transpiler. You can see an example at https://github.com/skiptools/skipapp-hello/blob/main/Package.swift

With this technique, you can either turn Skip off by editing the value of the flag directly in the `Package.swift` file, or, if you want to do it on a per-process level, you can quit Xcode and then re-launch it with the environment set, like:

```
SKIP_ZERO=1 xed /path/to/Darwin/Project.xcodeproj
```
